modules/infinite-scroll/infinity.php

```diff
@@ -1679,6 +1692,182 @@ public function add_mejs_config( $scripts_data ) {
 		}
 		return $scripts_data;
 	}
+
+	/**
+	 * Load AMP specific hooks.
+	 *
+	 * @return void
+	 */
+	public function amp_load_hooks() {
+		if ( Jetpack_AMP_Support::is_amp_request() ) {
```

Comment:
We should check if class_exists here as per previous instruction on some of the other issues we've worked on

Reply:
Addressed.